src/src/targets/EWRF_E7082VM/EWRF_E7082VM.c

```diff
@@ -63,6 +63,30 @@ uint8_t amp_state = 0;
                                       { 820,   830,  830,  840,  860,  890,  890,  895,  895}, // 100 mW
                                       {1450,  1450, 1450, 1450, 1450, 1450, 1450, 1450, 1450}  // ~400 mW
                                       };
+#elif defined(BETAFPV_A03)
+  #define CAL_FREQ_SIZE 9
+  #define CAL_DBM_SIZE 17
+  uint16_t calFreqs[CAL_FREQ_SIZE] = {5600,	5650,	5700,	5750,	5800,	5850,	5900, 5950, 6000};
+  uint8_t calDBm[CAL_DBM_SIZE] = {10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26};
+  uint16_t calVpd[CAL_DBM_SIZE][CAL_FREQ_SIZE] = {
```

Comment:
Can you show us the calibrated output power measurement results?

Reply:
[A03log_20220708.txt](https://github.com/OpenVTx/OpenVTx/files/9082636/A03log_20220708.txt)
[A03VpdValue.xlsx](https://github.com/OpenVTx/OpenVTx/files/9082637/A03VpdValue.xlsx)
[SpectrumLog_20220708.txt](https://github.com/OpenVTx/OpenVTx/files/9082638/SpectrumLog_20220708.txt)
These are test files. [A03log_20220708.txt] records the data read from A03, [SpectrumLog_20220708.txt]is the data read from the spectrometer, and [A03VpadValue.xlsx] is the data sorted from 1 and 2 according to the timestamp

Reply:
[A03VpadValue.xlsx] is the data sorted from [A03log_20220708.txt] and  [SpectrumLog_20220708.txt] according to the timestamp

Reply:
@JyeSmith @cruwaller  Are there any other questions here?